client/scripts/controllers/server/comments.ts

```diff
@@ -162,6 +163,26 @@ class CommentsController {
     }
   }
 
+  public async delete(comment) {
+    const _this = this;
+    return new Promise((resolve, reject) => {
+      // TODO: Change to DELETE /comment
+      $.post(`${app.serverUrl()}/deleteComment`, {
+        jwt: app.user.jwt,
+        comment_id: comment.id,
+      }).then((result) => {
+        const existing = this._store.getById(comment.id);
+        this._store.remove(existing);
+        // app.recentActivity.removeAddressActivity([comment]);
```

Comment:
Can we add a comment on why this is commented out/not possible?

Reply:
Aye-aye